test/Dockerfile.test.stretch

```diff
@@ -0,0 +1,9 @@
+FROM {image}
```

Comment:
Would it make sense to rename the existing Dockerfiles `Dockerfile.test.windows` and `Dockerfile.test.linux`?  It would eliminate this file which currently appears identical to the jessie Dockerfile and would simplify some of the `$image_os` logic in `test.ps1`